roles/bootstrap-os/tasks/bootstrap-fedora.yml

```diff
@@ -40,6 +40,8 @@
 # See https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#managed-node-requirements
 - name: Install libselinux-python3
   package:
-    name: libselinux-python3
+    # name: libselinux-python3
+    name: python3-libselinux
     state: present
+    use: dnf
```

Comment:
Without this, the install has not worked.

Reply:
can you elaborate why `boostrap-fedora.yml` is called? With  `fedora coreos` this yml should not not be called but `bootstrap-fedora-coreos.yml`. 
for `fedora-coreos`:
```
- include_tasks: bootstrap-fedora-coreos.yml
  when: '"ID=fedora" in os_release.stdout and "VARIANT_ID=coreos" in os_release.stdout'
..
- include_tasks: bootstrap-fedora.yml
  when:
    - '"Fedora" in os_release.stdout'
    - '"VARIANT_ID=coreos" not in os_release.stdout'
```

Reply:
Interesting question. I'm afraid that the answer it: It from my very first tries: At the very beginning, I tried to install kube on my fedora 32 host system (as master node) and on 2 libvirt fedora coreos VMs (as workers). Hence it is a problem, but wrong in this PR?!?

Reply:
if not any job fails with, im fine with..it may also be a fedora 32/ansible version support thing.
from ansible `package`:
> use: The default 'auto' will use existing facts or try to autodetect it
and fedora 32 is not yet tested/supported by kubespray.